shared/chat/conversation/list-area/normal/container.js

```diff
@@ -53,6 +45,30 @@ let markedInitiallyLoaded = false
 
 export default compose(
   connect(mapStateToProps, mapDispatchToProps, mergeProps),
+  withStateHandlers(
+    {
+      _conversationIDKey: null,
+      _lastOrdinalTime: Date.now(),
+    },
+    {
+      // We don't let you try and load more within a second. Used to use the ordinal but maybe we just never want a super quick load
```

Comment:
native and desktop both had ways to handle calling the loadMoreMessages too often, instead we just have a simple time based bookkeeping here